lib/src/config/object_store.rs

```diff
@@ -111,6 +169,34 @@ impl TryFrom<Table> for ObjectStoreConfig {
                     .expect("Lock was not poisoned");
             }
         }
+
         Ok(ObjectStoreConfig(obj_store))
     }
 }
+
+fn read_json_contents(file: &Path) -> Result<HashMap<String, String>, ObjectStoreConfigError> {
```

Comment:
This is mostly just copy-pasted from the dictionary implementation with different errors